<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QBRSettingsDialog</class>
 <widget class="QDialog" name="QBRSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>444</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Menu and toolbar</string>
   </property>
   <widget class="QRadioButton" name="uiMenuAndToolbar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>511</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Menu and toolbar</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="uiMenuOnly">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>511</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Only menu</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="uiToolbarOnly">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>511</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Only toolbar</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="statusBarEnable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>521</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Show statusbar</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enableLoadLast">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>521</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Load last file on start</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>541</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Book Design</string>
   </property>
   <widget class="QPushButton" name="bgColorButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bgColorLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Background color</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QBRSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QBRSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bgColorButton</sender>
   <signal>clicked()</signal>
   <receiver>QBRSettingsDialog</receiver>
   <slot>bgColorChoose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>bgColorChoose()</slot>
 </slots>
</ui>
